app/controllers/about_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
     @smart_proxies = SmartProxy.authorized(:view_smart_proxies).includes(:features)
     @compute_resources = ComputeResource.authorized(:view_compute_resources)
     @plugins = Foreman::Plugin.all
+    @releases = Wiki.new.versions if Setting[:released_versions]
```

Comment:
IMHO this should be a seperate AJAX request, so it does not block foreman.
further, I beleive that sometimes foreman itself wont allow connection to the internet, but maybe the user browser can, which in this case becomes a client side only implementation. I'm fine with this as an initial approach.